migrations/sql/V2019.02.26.9.21__alter_party_table_rename_address_columns.sql

```diff
@@ -0,0 +1,46 @@
+ALTER TABLE party
+    ADD COLUMN address_type_code varchar(3) DEFAULT 'CAN' NOT NULL;
+
+ALTER TABLE party RENAME province_code TO region_code;
+ALTER TABLE party RENAME postal_code   TO post_code;
+ALTER TABLE party
+    RENAME CONSTRAINT "party_province_code_fkey" TO "party_region_code_fkey";
+
+CREATE TABLE IF NOT EXISTS address_type_code (
+    address_type_code character varying(3)                   NOT NULL,
+    description       character varying(50)                  NOT NULL,
+    display_order     integer                                        ,
+    active_ind        boolean                  DEFAULT true  NOT NULL,
+    create_user       character varying(60)                  NOT NULL,
+    create_timestamp  timestamp with time zone DEFAULT now() NOT NULL,
+    update_user       character varying(60)                  NOT NULL,
+    update_timestamp  timestamp with time zone DEFAULT now() NOT NULL
+);
+
+ALTER TABLE address_type_code OWNER TO mds;
+COMMENT ON TABLE address_type_code IS 'Lookup table for valid address types';
```

Comment:
Just a picky thing, but I can see how this code table might be confusing to some... can we update the table description to "Lookup table for valid address types, used to decipher a range of addressing styles used in different countries."  (or something like that)

Reply:
Yep. I'll copy/paste that description directly